tests/new_ci_collation/data/test1.sql

```diff
@@ -0,0 +1,38 @@
+drop database if exists `new_ci_collation_test`;
+create database `new_ci_collation_test`;
+use `new_ci_collation_test`;
+
+CREATE TABLE t1 (
+    a varchar(20) charset utf8mb4 collate utf8mb4_general_ci primary key,
+    b int default 10
+);
+
+CREATE TABLE t2 (
+    id int primary key auto_increment,
+    a varchar(20) charset utf8mb4 collate utf8mb4_general_ci,
+    b int default 10
+);
+
+CREATE TABLE t3 (
+    a int primary key,
+    b varchar(10) collate utf8mb4_general_ci,
+    c varchar(10) not null,
+    unique key c(c)
+);
+
+CREATE TABLE t4 (
+    a varchar(10) charset utf8mb4 collate utf8mb4_bin, primary key(a)
+);
+
```

Comment:
Could you test `charset utf8 collate utf8_general_ci` too?

https://docs.pingcap.com/tidb/stable/character-set-and-collation#new-framework-for-collations

Reply:
added, PTAL